app/interactors/send_timer_reminders.rb

```diff
@@ -0,0 +1,18 @@
+class SendTimerReminders
+  include Interactor
+  include HasHarvest
+
+  before do
+    context.date ||= Date.current
+  end
+
+  def call
+    User.all.each do |user|
+      next if user.timer_reminder_sent_on.try(:today?)
```

Comment:
I guess I disagree. This is a very simple query (the field name makes it long) and even though we're dealing with a small number of users, alarms went on in my head immediately when I saw both the `.all` and the checking in Ruby. 

If you don't like the null check, remove the null and set an initial date to `created_at` or something.